NervanaSystems/tensorboard
# Added missing tests

Prerequisites for running tests (for your tensorflow Virtualenv):
- latest tensorflow and tensorboard, I am using 1.8.0 now
`pip install --upgrade tensorflow tensorboard`
- Install grpc and grpcio
`pip install grpc grpcio`

How to test:
- Python tests:
`bazel test tensorboard/...`

- Selenium:
`bazel test //tensorboard/functionaltests:core_test`

- Frontend unit tests related to Graph plugin changes:
`bazel run //tensorboard/plugins/graph/tf_graph_common/test`
open http://localhost:6006/tf-graph-common/test/index.html and `command + option + j` (if you are using Chrome) to see if it passed.
